cpp/ps0000/cpp_ps0000.cpp

```diff
@@ -1,7 +1,8 @@
-#include <iostream>
+#include <iostream> // 2/2
 using namespace std;
-int Main()
+int Main() // 0/1
 {
-  cout << "Hello World"<<"\n";
-  return 0;
-}
\ No newline at end of file
+  cout << "Hello, World\n"; // 3/4

```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [x] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [ ] output is as required `Hello, World!` in a single line